### ISSUE TITLE
Gargoyle flying animation still shows when moving if disabled

### ISSUE THREAD

**Issue by calemccammon** [Hotride/OrionUO#124]
April 5, 2020 build
OSI 7.0.85.14

1. Launch UO and sign in with a Gargoyle
2. Options > Uncheck flying animation
3. Restart UO (in the original client a restart is required)
4. Sign in with your Gargoyle
5. Move your character
6. Observe character moves with flying animation

**Comment by Hotride** [comment]
Yes, this option does not work yet, because it’s not clear whether this disables the flight animation only for gargoyles, or in general for all creatures.
